libsolidity/formal/SMTChecker.cpp

```diff
@@ -272,17 +270,13 @@ bool SMTChecker::visit(ForStatement const& _node)
 	auto indicesBeforeLoop = copyVariableIndices();
 
 	// Do not reset the init expression part.
-	auto touchedVariables =
-		m_variableUsage->touchedVariables(_node.body());
+	auto touchedVars = touchedVariables(_node.body());
 	if (_node.condition())
-		touchedVariables += m_variableUsage->touchedVariables(*_node.condition());
+		touchedVars += touchedVariables(*_node.condition());
 	if (_node.loopExpression())
-		touchedVariables += m_variableUsage->touchedVariables(*_node.loopExpression());
-	// Remove duplicates
```

Comment:
`touchedVars` is a `set` now instead of a `vector`

Reply:
Yep - first time I felt with @chriseth complaining about ``auto`` :-)